app/components/throbber.html.erb

```diff
@@ -0,0 +1 @@
+<div class="throbber"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
```

Comment:
*blinks*

Reply:
@jcoyne 	but srsly, what's going on here? :smiley:

Reply:
Those divs are the "dots" in the animated throbber